src/build/content/prerendered.ts

```diff
@@ -195,7 +207,41 @@ export const copyPrerenderedContent = async (ctx: PluginContext): Promise<void>
               await writeCacheEntry(key, value, lastModified, ctx)
             }),
         ),
-      )
+        ...Object.entries(manifest.dynamicRoutes).map(async ([route, meta]) => {
+          // fallback can be `string | false | null`
+          //  - `string` - when user use pages router with `fallback: true`, and then it's html file path
+          //  - `null` - when user use pages router with `fallback: 'block'` or app router with `export const dynamicParams = true`
+          //  - `false` - when user use pages router with `fallback: false` or app router with `export const dynamicParams = false`
+          if (typeof meta.fallback === 'string') {
+            // https://github.com/vercel/next.js/pull/68603 started using route cache to serve fallbacks
+            // so we have to seed blobs with fallback entries
+
+            // create cache entry for pages router with `fallback: true` case
+            await limitConcurrentPrerenderContentHandling(async () => {
+              // dynamic routes don't have entries for each locale so we have to generate them
+              // ourselves. If i18n is not used we use empty string as "locale" to be able to use
+              // same handling wether i18n is used or not
+              const locales = ctx.buildConfig.i18n?.locales ?? ['']
+
+              const lastModified = Date.now()
+              for (const locale of locales) {
+                const key = routeToFilePath(posixJoin(locale, route))
+                const value = await buildPagesCacheValue(
+                  join(ctx.publishDir, 'server/pages', key),
+                  shouldUseEnumKind,
+                  true, // there is no corresponding json file for fallback, so we are skipping it for this entry
+                )
+                // Netlify Forms are not support and require a workaround
+                if (value.kind === 'PAGE' || value.kind === 'PAGES' || value.kind === 'APP_PAGE') {
+                  verifyNetlifyForms(ctx, value.html)
+                }
```

Comment:
Yeah, I'll drop that completely form here - we actually already are doing this right now because of https://github.com/netlify/next-runtime/blob/5f5ec0651726abf1027319707beaa1580a0c5dd8/src/build/content/static.ts#L35 so this was just doing same check again on same content

This was just copied from above code heh